### PR TITLE
fixing radius of controller near interaction orb

### PR DESCRIPTION
**Pull request by rickfromwork**
The radius of the orb was not tied to the radius of the collision sphere, which created some visual discrepancy.

### CODE REVIEW

File: src/XR/features/WebXRNearInteraction.ts [hunk 1]
```diff
@@ -791,23 +786,33 @@ export class WebXRNearInteraction extends WebXRAbstractFeature {
         const easingFunction = new QuadraticEase();
         easingFunction.setEasingMode(EasingFunction.EASINGMODE_EASEINOUT);
 
+        const hoverSizeVec = new Vector3(this._controllerPickRadius, this._controllerPickRadius, this._controllerPickRadius);
```
Comment (sebavan) [inline src/XR/features/WebXRNearInteraction.ts:789]: All the numbers sounds a bit magical, wondering if we should comment on their provenance ?

Comment (rickfromwork) [reply src/XR/features/WebXRNearInteraction.ts:789]: I made the fractions based off of the float values that were in place before, I'll add in descriptors though.

**Review by RaananW — APPROVED** [review]
There is a build issue, but LGTM otherwise

**Comment by sebavan** [comment]
@rickfromwork it should be all good if you pull master in

**Comment by RaananW** [comment]
@rickfromwork - any update on this one?

**Comment by rickfromwork** [comment]
I cry. This PR slipped my mind last week.

File: src/XR/features/WebXRNearInteraction.ts [hunk 2]
```diff
@@ -791,23 +786,36 @@ export class WebXRNearInteraction extends WebXRAbstractFeature {
         const easingFunction = new QuadraticEase();
         easingFunction.setEasingMode(EasingFunction.EASINGMODE_EASEINOUT);
 
+        // Adjust the visual size based off of the size of the touch collision orb.
+        // Having the size perfectly match for hover gives a more accurate tell for when the user will start interacting with the target
+        // Sizes for other states are somewhat arbitrary, as they are based on what feels nice during an interaction
```
Comment (sebavan) [inline src/XR/features/WebXRNearInteraction.ts:791]:
```suggestion
        // Sizes for other states are somewhat arbitrary, as they are based on what feels nice during an interaction
```

**Pull request merged**